### PR TITLE
Added disbursement support and unit tests

### PR DESCRIPTION
**Pull request by ngo275**

### CODE REVIEW

File: skeleton-model-app/market/utils/composeDisbursementClaims.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      commitmentKey = `${BNPL_ORDER}:${orderId}:${key}:${COMMITMENT}`;
+    }
+
+    claims[`${BNPL_ORDER}:${orderId}:${key}`] = {
+      [`${BNPL_ORDER}:${orderId}:${key}`]: {
+        content_type: "pdf",
+        content: documentContent,
+      },
+      // TODO: fix
+        // [commitmentKey]: commitments ? commitments[claimKey] : undefined,
```
Comment (danhnn) [inline skeleton-model-app/market/utils/composeDisbursementClaims.js:49]: @ngo275 the code in line 49 is correct so why do we need to fix it? The commitmentKey is required here, just its value can be undefined or equal to commitments[claimKey]  depend on withCommitments param. The logic here is the same with composeUserClaim, correct ?

File: skeleton-model-app/market/dataInteraction/index.js [hunk 2]
```diff
@@ -356,10 +362,82 @@ const handleOrder = async (db, id, payload) => {
...
 }
 
+/**
+ * This function retrieves order commitment values from DB
+ *
+ * @param db
+ * @param orderId
+ * @returns {Promise<Object|null>}
+ */
+const fetchOrderCommitment = async (db, orderId) => {
```
Comment (danhnn) [inline skeleton-model-app/market/dataInteraction/index.js:378]: @ngo275 I don't see `saveOrderCommitment` method anywhere in the source code so how can we get the order's commitment?

Comment (ngo275) [reply skeleton-model-app/market/dataInteraction/index.js:378]: I added a column in order table.

File: skeleton-model-app/market/handlers/pushDisbursementClaims.js [hunk 3]
```diff
@@ -13,51 +16,34 @@ const pushDisbursementClaims = async (req, res, { db, credify }) => {
...
 
     const commitments = await credify.claims.push(
       organizationId,
       credifyId,
       claims
     )
 
+    console.log("test")
+
     await upsertCommitments(db, credifyId, commitments);
```
[the remark this answers is not in the document]
Comment (ngo275) [reply skeleton-model-app/market/handlers/pushDisbursementClaims.js:43]: Yeah, this was a mistake. Fixed.

**Pull request merged**